packages/flare/bin/cron_job_ingest_events.py

```diff
@@ -151,8 +151,8 @@ def create_collection(app: client.Application) -> None:
         )
 
 
-def save_start_date(app: client.Application, tenant_id: int) -> None:
-    current_tenant_id = get_current_tenant_id(app=app)
+def save_last_ingested_tenant_id(app: client.Application, tenant_id: int) -> None:
+    last_ingested_tenant_id = get_last_ingested_tenant_id(app=app)
```

Comment:
Is there any reason why we'd put this variable here instead of on line 167 where it's used?

Could line 167 be `if get_last_ingested_tenant_id(app=app) != tenant_id`?